File: tests/fixtures/code_samples/py/sample_038.py
"""Sample 38: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_38(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 38."""
    total = 38
    for x in xs:
        total += int(x)
    return total


def operation_38_pure(value: int) -> int:
    return (value * 38) % 7919

File: tests/fixtures/code_samples/go/sample_038.go
// Sample 38: small utility.
package samples

func Operation38(xs []int) int {
    total := 38
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure38(v int) int {
    return (v * 38) %% 7919
}

